### ISSUE TITLE
Implement power flow modelling with passive homes

### ISSUE THREAD

**Issue by floracharbo** [floracharbo/MARL_local_electricity#35]

**Comment by julie-vienne** [comment]
@floracharbo I will tackle this once we have merged the add_reactive_power PR with main :)

**Comment by julie-vienne** [comment]
@floracharbo I would like to change the default values here to unify our tolerances across the different methods:

```
tol_rel_voltage_diff: 1.e-2
tol_rel_voltage_costs: 1.e-2
tol_abs_line_losses: 1.e-2
tol_voltage_iteration: 0.001 
```
I would suggest a value of 0.001 for the voltage differences and 0.01 for the voltage costs and line losses.

**Comment by floracharbo** [comment]
> tol_rel_voltage_diff

did you already implement this? this is what it looks like in the main branch already.

**Comment by julie-vienne** [comment]
> > tol_rel_voltage_diff
> 
> did you already implement this? this is what it looks like in the main branch already.

hmm I don't perfectly remember but my point was mainly about changing a default value (from 0.01 to 0.001 for `tol_rel_voltge_diff`. If I remember correctly, you wanted us to discuss this every time we want to update the defaults :)

**Comment by floracharbo** [comment]
oh sorry I misunderstood your point above. So I am just switching tol_rel_voltage_diff 1e-2 -> 1e-3.

**Comment by julie-vienne** [comment]
Yes, I think that would be good to unify both methods (subset of lines and iteration) but I haven't run a lot of experiments with this yet. We might have to have a larger tolerance in some cases, e.g. pf > 0.95.
